internal/cmd/trustpolicy/remote/remote.go

```diff
@@ -12,6 +12,10 @@ func New() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "remote",
 		Short: "Tools for managing remote policies",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			err := cmd.Flags().SetAnnotation("signing-key", cobra.BashCompOneRequiredFlag, []string{"false"})
```

Comment:
Can this instead happen when the child command is added to the parent? https://github.com/gittuf/gittuf/blob/main/internal/cmd/trust/trust.go#L24 and the equivalent for `policy` command, basically. That way, it happens once rather than on prerun every time the command is invoked.

Reply:
Done!